### PR TITLE
new package ddd

### PR DESCRIPTION
**Pull request by DarylGrunau**
This PR adds new package ddd.  It is dependent on the merge of PR #15703

### CODE REVIEW

File: var/spack/repos/builtin/packages/ddd/package.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    def configure_args(self):
+        spec = self.spec
+
+        args = [
+            '--disable-debug',
+            '--disable-dependency-tracking',
+            '--enable-builtin-app-defaults',
+            '--enable-builtin-manual',
+            '--enable-shared' if '+shared' in spec else '--disable-shared',
+            '--enable-static' if '+static' in spec else '--disable-static',
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ddd/package.py]: These variants are missing from the package.

Comment (DarylGrunau) [reply var/spack/repos/builtin/packages/ddd/package.py]: Fixing; standby-1

**Comment by adamjstewart** [comment]
Rebooting Travis...

**Comment by adamjstewart** [comment]
Flake8:
```
var/spack/repos/builtin/packages/ddd/package.py:63: [E124] closing bracket does not match visual indentation
var/spack/repos/builtin/packages/ddd/package.py:69: [E127] continuation line over-indented for visual indent
var/spack/repos/builtin/packages/ddd/package.py:71: [E124] closing bracket does not match visual indentation
var/spack/repos/builtin/packages/ddd/package.py:74: [N802] function name '_rename_exe_on_OSX' should be lowercase
```

**Pull request merged**